spec/query_spec.cr

```diff
@@ -474,27 +477,15 @@ describe Avram::Query do
       scores.each { |score| UserBox.create &.average_score(score) }
       query_sum = UserQuery.new.average_score.select_sum
       query_sum.should eq sum
-    end
-
-    it "works with chained where clauses" do
-      scores = [100.4, 123.22]
-      sum = scores.sum
-      out_of_range = 200
-      scores.each { |score| UserBox.create &.average_score(score) }
-      query_sum = UserQuery.new.average_score.lt(out_of_range).average_score.select_sum
-      query_sum.should eq sum
-    end
```

Comment:
I think this test would still be nice to have but maybe there is a reason for deleting it? Is it covered by another test?

Reply:
Yes, it's covered in a different test. I created a generic `describe "#select_sum" do` that handles the 2 non type specific tests here: https://github.com/luckyframework/avram/blob/69259484eac9995277a3384a6ac99d981ac09d26/spec/query_spec.cr#L402-L415

It seemed unnecessary to repeat for every type of integer and float. Let me know if you would like me to do something a little different.

Reply:
Ah I see! I think this was a good call 👍